notebooks/example_container_fmriprep_4review.yaml

```diff
@@ -0,0 +1,67 @@
+# This is an example config yaml file for fMRIPrep version 20.2.3.
+
+# Warning!!!
+#   This is only an example, which may not necessarily fit your purpose,
+#   or be an optimized solution for your case,
+#   or be compatible to the fMRIPrep version you're using.
+#   Therefore, please change and tailor it for your case before use it!!!
+
+# Arguments when executing the BIDS App using `singularity run`:
+# You should not have flags of : `--participant-label`, or `--bids-filter-file`!
+babs_singularity_run:
+    -w: "$BABS_TMPDIR"   # this is a placeholder. To be changed to `${PWD}/.git/tmp/wkdir`
+    --n_cpus: '1'
+    --stop-on-first-crash: ""
+    --fs-license-file: "$FREESURFER_LICENSE" # this is a placeholder.
+    --skip-bids-validation: Null  # Null or NULL is also a placeholder
```

Comment:
Yes, `Null` and `""` both enable the flag without value, i.e., just `--stop-on-first-crash`. 

We were thinking about having all possible arguments in an example yaml file, but it would be a lot of responsibility regarding correctness etc, so we don't plan to do that.

Reply:
True, but I think that labs that use babs are likely to have their own internal templates that individual lab members modify, rather than writing new files from scratch. In that case, it makes sense for them to have all of the parameters they _might_ use for a given BIDS App, which grad students would copy and change for their own purposes.